.github/workflows/publish-to-maven.yml

```
@@ -0,0 +1,62 @@
+name: Release to Maven Central
+on:
+  push:
+
+jobs:
+
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: "21"
+
+      - name: Get New Version
+        id: generate-new-tag
+        uses: anothrNick/github-tag-action@1.73.0
+        env:
+          TAG_PREFIX: crypt4gh-
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DRY_RUN: true
+          DEFAULT_BUMP: patch
+          MAJOR_STRING_TOKEN: "#major_crypt4gh"
+          MINOR_STRING_TOKEN: "#minor_crypt4gh"
+          PATCH_STRING_TOKEN: "#patch_crypt4gh"
+      - name: Extract New Version
+        id: new-tag
+        run: |
+          full_version="${{ steps.generate-new-tag.outputs.new_tag }}"
+          component="crypt4gh"
+          version="${full_version#"$component"-}"
+          echo "version=$version" >> "$GITHUB_OUTPUT"
+
+      - name: Build and publish to local staging
+        env:
+          SIGNING_KEY_BASE64: ${{ secrets.SIGNING_KEY_BASE64 }}
+          SIGNING_PASSWORD: ${{ secrets.SIGNING_PASSWORD }}
+        run: |
+          ./gradlew lib:crypt4gh:clean lib:crypt4gh:build lib:crypt4gh:publishAllPublicationsToLocalStagingRepository \
```

```suggestion
          ./gradlew lib:crypt4gh:clean lib:crypt4gh:build lib:crypt4gh:publishMavenJavaPublicationToLocalStagingRepository \
```
  The MavenJava publication includes all the stuff that needs to be publised to Maven Central (and more). Including the "library" and "tool" publications here as well (with "AllPublications") will be redundant and may even mess things up.